packages/catalog-process/src/services/catalogService.ts

```diff
@@ -3144,7 +3140,7 @@ export async function createOpenApiInterfaceByTemplate(
           contentType,
           checksum,
           serverUrls,
-          interfaceTemplateMetadata: apiContactsData,
+          interfaceTemplateMetadata: { ...eserviceInstanceInterfaceRestData },
```

Comment:
```suggestion
      interfaceTemplateMetadata: eserviceInstanceInterfaceRestData,
```

Reply:
Also you could call the param of `verifyAndCreateDocument` in the same way, so maybe here you can just pass `eserviceInstanceInterfaceRestData` as a shorthand property name